debian/control

```diff
@@ -3,7 +3,7 @@ Section: admin
 Priority: optional
 Homepage: https://cloud-init.io/
 Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
-Build-Depends: debhelper (>= 9.20160709),
+Build-Depends: debhelper-compat (= 13),
```

Comment:
is this ok going from a > to an explicit equal? This means we'll have to change each dev release potentially vs not in the past?

Reply:
Should be OK as newer releases also provide the older compat levels:
```
$ apt show debhelper
...
Provides: debhelper-compat (= 9), debhelper-compat (= 10), debhelper-compat (= 11), debhelper-compat (= 12), debhelper-compat (= 13), ...
```

Reply:
It's actually necessary to use a strict equal when using `debhelper-compat` to declare the compat level, otherwise debhelper fails:

```
dh clean --with python3 --buildsystem pybuild
dh: warning: Found invalid debhelper-compat relation: debhelper-compat (>= 13)
dh: warning:  * Please format the relation as (example): debhelper-compat (= 13)
dh: warning:  * Note that alternatives, architecture restrictions, build-profiles etc. are not supported.
dh: warning:  * If this is not possible, then please remove the debhelper-compat relation and insert the
dh: warning:    compat level into the file debian/compat.  (E.g. "echo 13 > debian/compat")
dh: error: Could not parse desired debhelper compat level from relation: debhelper-compat (>= 13)
```

Reply:
ahh thanks for this @paride, I had the same question